overview/intro.md

```diff
@@ -2,67 +2,113 @@
-6. Multiple options on formats (display, video, native) & channels (mobile, desktop)
-7. Helpful community with lots of free setup advice, as well as professional setup & services
+![Prebid.js Simple Flowchart](/assets/images/flowcharts/pb-js-simple.png) 
 
-## How Does Prebid Work?
+### Prebid Server
 
-At a high level, header bidding involves just a few steps:
+Prebid Server provides a server side solution to header bidding. Built on the same core principles of Prebid.js, our server solution can reduce latency and improve page load time.
+Several Prebid.org members provide hosted solutions, enabling publishers to receive the benefits of server side header bidding without the need to implement and manage the process themselves. 
 
-1. The ad server's tag on page is paused, bound by a timer, while the Prebid.js (or SDK) library fetches bids and creatives from various SSPs & exchanges you want to work with.
-1. Prebid.js (or SDK) passes information about those bids (including price) to the ad server's tag on page, which passes it to the ad server as query string parameters.
-1. The ad server has line items targeting those bid parameters.
-1. If the ad server decides Prebid wins, the ad server returns a signal to Prebid.js (or SDK) telling the library to write the winning creative to the page. All finished!
+If a publisher would prefer to implement their own solution, [source code](https://github.com/prebid/prebid-server) is available from our Github page and detailed instructions for configuring, deploying and testing your implementation can be found in the [Prebid Server section](/prebid-server/prebid-server-overview.html) of thi site.  
 
-![Ad Ops Diagram]({{ site.github.url }}/assets/images/adops-intro.png){: .pb-lg-img :}
+**Prebid Server process**
 
-## Further Reading
+Prebid Server provides multiple endpoints for auctions as well as data retrieval and supports the AMP (accelerated mobile pages) format. The primary endpoint is `/openrtb2/auction` and the process follows these steps: 
 
+1. An OpenRTB request is sent to Prebid Server. 
+2. Prebid Server conducts auction with selected SSPs and ad exchanges. 
+3. PBS collects the bids and creatives and stores them in Prebid cache. 
+4. PBS passes the winning bid parameters to the ad server. 
+5. The ad server takes the passed in key-values and finds a matching line item and then compares to  other line items that bid on this impression. 
+6. If the ad server determines Prebid wins the auction it returns a reference to the winning creative to Prebid. 
+7. Prebid retrieves the winning creative and displays it in the correct ad slot. 
+
+![Prebid Server Flowchart](/assets/images/flowcharts/pb-server-display.png) 
+
+### Prebid Mobile
+
+For mobile apps, Prebid provides Prebid Mobile (PBM), an end-to-end header bidding solution for both iOS and Android. Working in conjunction with Prebid Server, PBM reduces latency and enables access to more mobile buyers, 
```

Comment:
typo: end with a period, not a comma

Reply:
Fixed, it actually was a bad break, not a sentence end. It reads now as a listing of benefits.